api/migrations/versions/0205_fix_email_formatting.py

```diff
@@ -0,0 +1,66 @@
+"""
+
+Revision ID: 0205
+Revises: 0204
+Create Date: 2018-11-12 15:32:20.317286
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0205'
+down_revision = '0204'
+
+
+templates = [{
+    'id': "4f46df42-f795-4cc4-83bb-65ca312f49cc",
+    'new_name': "Notify invitation email",
+    'new_subject': "((user_name)) has invited you to collaborate on ((service_name)) on Notify.gov.au",
```

Comment:
I'm no copy expert. but this is the same length and reads worse than our content

collaborate ON x ON notify.gov

Reply:
I didn't change it though. Only fixed the preformatted text issue.

Reply:
Good pickup. Well, kind of. I don't actually update the subject in this migration so will remove the keys from the dict.